src/components/folio-number-input/FolioNumberInput.vue

```diff
@@ -49,6 +51,27 @@ export default class FolioNumberInput extends Vue {
     ]
   }
 
+  /** Called when component is created. */
+  created (): void {
+    this.folioNumberString = this.folioNumber
+  }
+
+  /** Emit folio number on change. */
+  @Watch('folioNumberString')
+  private onFolioNumberChange (): void {
+    this.emitFolioNumber(this.folioNumberString)
+  }
```

Comment:
I am OK with this, if we can't get `update:modelValue` working soon.

Reply:
I removed the `folioNumberString` watcher since we got the `update:folioNumberString` working.